src/nnet3bin/nnet3-align-compiled.cc

```diff
@@ -69,8 +69,6 @@ int main(int argc, char *argv[]) {
                 "yes|no|optional|wait, only has effect if compiled with CUDA");
     po.Register("transition-scale", &transition_scale,
                 "Transition-probability scale [relative to acoustics]");
-    po.Register("acoustic-scale", &acoustic_scale,
```

Comment:
Right. Done and cleaned.


在 2016/6/17 1:53, Daniel Povey 写道:
>
> In src/nnet3bin/nnet3-align-compiled.cc 
> <https://github.com/kaldi-asr/kaldi/pull/850#discussion_r67392792>:
>
> > @@ -69,8 +69,6 @@ int main(int argc, char *argv[]) {
> >                  "yes|no|optional|wait, only has effect if compiled with CUDA");
> >      po.Register("transition-scale", &transition_scale,
> >                  "Transition-probability scale [relative to acoustics]");
> > -    po.Register("acoustic-scale", &acoustic_scale,
>
> please remove this variable also, it seems to be unused.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/850/files/5b6ea2544905245b1557282c29fdcd3f47c5a7d3#r67392792>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ADKpxNI7DhyQRtAaZFTyBkpS7k-lXvmlks5qMY2qgaJpZM4I3G_5>.
>